xml/depl_require.xml

```diff
@@ -2069,6 +2069,24 @@
         between the cluster nodes) to ensure redundancy.
        </para>
       </important>
+      <important>
+       <title>Dedicated networks</title>
+       <para>
+        The <literal>corosync</literal> network communication layer is
```

Comment:
But you helped me realise that actually we *force* corosync traffic to go over the admin network.  So I'll reword it to make that clearer.